buildSrc/src/main/java/Dependencies.kt

```diff
@@ -7,9 +7,8 @@ object Versions {
     const val targetSdkVersion = 28
     const val versionCode = 1
     const val versionName = "1"
-    const val gradleBuildTool = "4.0.1"
-    const val dataBinding = "3.2.0-alpha11"
-    const val kotlinVersion = "1.4.10"
+    const val gradleBuildTool = "4.1.1"
+    const val kotlinVersion = "1.3.21"
```

Comment:
Dependency version upgrades/downgrades are unnecessary in this PR. Better to have a different issue/PR for that.